backend/variants/tests/helpers.py

```
@@ -155,7 +155,7 @@ def run_count_query(
             return result
 
 
-class TestViewsBase(TestCase):
+class ViewTestBase(TestCase):
```

Ruff

Redefinition of unused `ViewTestBase` from line 47

(F811)